library/src/morpheus-master/sass/modules/tool/profile2/_profile2.scss

```diff
@@ -1,10 +1,1224 @@
 .#{$namespace}sakai-profile2 {
 
+	.profile-twitter-account td.label{
```

Comment:
I don't recommend reviewing the first portion file, I've left the original profile2.css file in this commit that contains the same changes as lines 3-1216 of this file as that is easier to see the difference in the refactor.

Lines 1217-1344 of this file are new changes that should be reviewed and are not contained elsewhere. I'll remove the old profile2.css file before merge.